app/helpers/payment_details_helper.rb

```diff
@@ -51,4 +51,17 @@ def payment_details_definition_list_item(name, account)
     definition_list_item += content_tag :dd, value.presence, class: 'u-dl-definition'
     definition_list_item
   end
+
+  def stripe_billing_address_json
+    billing_address = current_account.billing_address
```

Comment:
OK I'll be the ruby PITA 😄 use a delegate instead
```suggestion
  delegate :billing_address, to: :current_account
```

Also while you're at it, add  at the top of the file:
```ruby
# frozen_string_literal: true
```

Reply:
What does `delegate` do?

Reply:
@josemigallas In fact, it fails with
```
undefined method `delegate' for #<#<Class:0x00007fcc5289cda0>:0x00007fcc6066a128>
```

Reply:
And if I move it outside the method it fails with
```
undefined local variable or method `current_account' for PaymentDetailsHelper:Module
```

Reply:
nvm, i found an article